app/views/bookmarks/_external_work_fields.html.erb

```diff
@@ -1,25 +1,9 @@
 <fieldset>
-	<legend><%= ts('External Work') %></legend>
+  <legend><%= ts('External Work') %></legend>
   <dl>
     <dt class="required"><%= ew.label :url, ts('URL') + '*' %></dt>
     <dd class="required" title="URL">
-      <% if !params[:url_from_external].blank? %>
-        <%= ew.text_field :url, autocomplete_options('external_work', :autocomplete_token_limit => 1, :value => params[:url_from_external]) %>
-      <% else %>
-        <%= ew.text_field :url, autocomplete_options('external_work', :autocomplete_token_limit => 1) %>
-      <% end %>
-      <%= hidden_field "fetched", "", :id => "fetched" %>
-      <%= content_for :footer_js do %>
-        <%= javascript_tag do %>
-          $j("#bookmark_external_url").blur(function() {
-            $j.ajax({
-              type: "get",
-              data: "external_work_url=" + $j("#bookmark_external_url").val(),
-              dataType: "script",
-              url: "<%= url_for(:controller => "external_works", :action => "fetch") %>"
-            })
-          })
-        <% end %>
+      <%= ew.text_field :url %>
```

Comment:
We still want to set the value of the URL field using the params from the bookmarklet, so I think we need to do:

```
<% if !params[:url_from_external].blank? %>
  <%= ew.text_field :url, value: params[:url_from_external] %>
<% else %>
  <%= ew.text_field :url %>
<% end %>
```